packages/protobuf/src/types/component.ts

```diff
@@ -0,0 +1,332 @@
+/**
+ * This file re-exports all types from
+ * `@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/component_name/v1/component_name_pb.js`
+ * except the duplicated conflicting types (e.g. `GenesisContent`).
+ */
```

Comment:
TypeScript is complaining if these types are star-exported like this: 

<img width="1182" alt="Screenshot 2024-07-29 at 15 46 30" src="https://github.com/user-attachments/assets/c242951b-912a-4129-a73f-6af6b328f2fa">

That's why I carefully re-exported non-duplicated types only